markdown/dbaas.md

```diff
@@ -334,12 +402,58 @@ curl -X POST "http://localhost/v1/management/DBaaS/PXCClusters/GetCredentials" \
 
 API endpoint used in this step: [GetPXCClusterCredentials](ref:getpxcclustercredentials)
 
+### Create a PSMDB Cluster
+
+The PSMDB Create endpoint can also set defaults so, creating a PSMDB cluster can be made with a request like this:
+
+```bash
+curl -X POST "http://localhost/v1/management/DBaaS/PSMDBCluster/Create" \
+    -H "accept: application/json" \
+    -H "authorization: Basic YWRtaW46YWRtaW4=" \
+    -H "Content-Type: application/json" \
+    -d "{ \"kubernetes_cluster_name\": \"my_cluster\", \"expose\": true}"
+```
+
+#### Request fields
+
+```json
+{
+  "kubernetes_cluster_name": "string",
+  "name": "string",
+  "params": {
+    "cluster_size": 0,
+    "replicaset": {
+      "compute_resources": {
+        "cpu_m": 0,
+        "memory_bytes": "string"
+      },
+      "disk_size": "string"
+    },
+    "image": "string"
+  },
+  "expose": true
+}
+```
+
+| Field                                     | Description                           | Notes                                                        |
+| ----------------------------------------- | ------------------------------------- | ------------------------------------------------------------ |
+| kubernetes_cluster_name                   | Kubernetes cluster name               | Required                                                     |
+| name                                      | PSMDB cluster name                    | Default: `psmdb`+DB version+4 digits random number           |
+| cluster_size                              | Cluster size                          | Default: 3                                                   |
+| replicaset.compute_resources.cpu_m        | CPU resources millis                  | Default: 1000                                                |
+| replicaset.compute_resources.memory_bytes | Max memory size in bytes              | Default: 2 Gb                                                |
+| disk_size                                 | Max disk size                         | Default: 25 Gb                                               |
+| image                                     | PSMDB Docker image                    | Default is the recommended version from the Percona's version service |
+| expose                                    | Expose outside the Kubernetes cluster | Default: false                                               |
```

Comment:
I think it should be included in https://percona-pmm.readme.io/reference/createpsmdbcluster
to do it you should create a file `createpsmdbcluster.md` in the `markdown` folder and put it there.

Reply:
Added new files for Create PXC and Create PSMDB